<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auto</class>
 <widget class="QDialog" name="Auto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="actionButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>자동예약</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>실패한 날짜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>요일</string>
   </property>
  </widget>
  <widget class="QListView" name="fail_list_view">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="mon_chk_box">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="tue_chk_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>화</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="wed_chk_box">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>수</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="thu_chk_box">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>목</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fri_chk_box">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>금</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sat_chk_box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>토</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sun_chk_box">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>일</string>
   </property>
  </widget>
  <widget class="QComboBox" name="time_combo_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>시간대</string>
   </property>
  </widget>
  <widget class="QListView" name="pass_list_view">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>571</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>성공한 날짜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="jungu1_chk_box">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>정구1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>테니스 장소</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="jungu2_chk_box">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>정구2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="jungu3_chk_box">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>정구3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="jungu4_chk_box">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>정구4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sinea1_chk_box">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>실내1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sinea2_chk_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>실내2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="yaw3_chk_box">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>야외3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="yaw4_chk_box">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>야외4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="yaw7_chk_box">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>야외7</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
